sdk/helper/certutil/certutil_test.go

```diff
@@ -411,25 +411,60 @@ func TestTLSConfig(t *testing.T) {
 
 		switch usage {
 		case TLSServer | TLSClient:
-			if len(tlsConfig.ClientCAs.Subjects()) != 1 || bytes.Compare(tlsConfig.ClientCAs.Subjects()[0], pcbut.CAChain[0].Certificate.RawSubject) != 0 {
-				t.Fatalf("CA certificate not in client cert pool as expected")
+			if tlsConfig.ClientCAs == nil {
```

Comment:
I think `ClientCAs` is just a copy of `RootCAs`, if present, but I'm happy to leave this as-is to validate it is indeed provisioned.